### PR TITLE
Feature add arrangement date time

### PR DESCRIPTION
**Pull request by BenZacs**
I already add arrangement Dat and Time but format for date time must be YYYY/MM/DD HH:MM:SS and input must be 
ex.  2020/12/30 18:30:00

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (bleachjade) [inline .idea/vcs.xml]: You should not commit the .idea file onto the repo I think

File: Kvent/views.py [hunk 2]
```diff
@@ -50,9 +50,11 @@ def create_event(request):
             location = form.data.get('location')
             short_description = form.data.get('short_description')
             long_description = form.data.get('long_description')
+            arrange_time = form.data.get('arrange_time')
+            print(arrange_time)
```
Comment (bleachjade) [inline Kvent/views.py]: delete the print statement that you use to test the code

File: Kvent/tests/test_views.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        event = create_event()
+        url = reverse('event-detail', args=(event.id,))
+        response = self.client.get(url)
+        self.assertContains(response, event.long_description)
+    
+    # def test_arrange_time(self):
+    #     event = create_event()
+    #     url = reverse('event-detail', args=(event.id,))
+    #     response = self.client.get(url)
+    #     self.assertContains(response, "Dec. 12, 2020, 6 p.m.")
```
Comment (bleachjade) [inline Kvent/tests/test_views.py]: delete this if you are not using it

File: Kvent/views.py (hunk 2 above)
Comment (BenZacs) [reply Kvent/views.py]: I already fix this part.

**Comment by bleachjade** [comment]
for the conflict part in create-event-page.html, I suggested that you should keep the form in the develop branch but add your arrangement time at the bottom of the HTML page @BenZacs

**Pull request merged**